common/xrdp_constants.h

```diff
@@ -190,13 +190,21 @@
 #define RDP_KEYPRESS                   0
 #define RDP_KEYRELEASE                 (KBD_FLAG_DOWN | KBD_FLAG_UP)
 
-#define MOUSE_FLAG_MOVE                0x0800
-#define MOUSE_FLAG_BUTTON1             0x1000
-#define MOUSE_FLAG_BUTTON2             0x2000
-#define MOUSE_FLAG_BUTTON3             0x4000
-#define MOUSE_FLAG_BUTTON4             0x0280
-#define MOUSE_FLAG_BUTTON5             0x0380
-#define MOUSE_FLAG_DOWN                0x8000
+/* Mouse Event (MS-RDPBCGR 2.2.8.1.1.3.1.1.3) */
+#define PTRFLAGS_HWHEEL                0x0400
+#define PTRFLAGS_WHEEL                 0x0200
+#define PTRFLAGS_WHEEL_NEGATIVE        0x0100
+#define WheelRorationMask              0x01FF
```

Comment:
Please follow the other defines styling (capitalizing and naming convention)

Reply:
I brought these constants as-is from MS-RDPBCGR. I think this is odd but using the same name as MS will be clear. What would be the best?
https://msdn.microsoft.com/en-us/library/cc240586.aspx

Reply:
I think the coding style should be consistent with the exisiting code, also there is a typo Roration -> Rotation

Reply:
I've only fixed the typo at this time. Actually it is not used anywhere. There's only definition so far.